### DISCUSSION TITLE
Can't create ability from "Using Replies" in wiki

### DISCUSSION THREAD

**Discussion by nikitashumsky** [rubenlagus/TelegramBots#1120] [Q&A]
Trying to repeat the code from "Using Replies" in wiki, but method .reply takes a BiConsumer as the first argument, so now I need to provide two arguments to lamba, should I? Because if I don't, it can't find .getMessage() method in my update.
![reply](https://user-images.githubusercontent.com/82108826/190237579-f8a8b7c6-2f23-4e34-bebf-69604e9c4f74.png)

**Comment by schkedy** [comment]
.reply use BiConsumer,  BiConsumer<BaseAbilityBot, Update> it means you have to initialize 2 arguments, now you have only upd, try that ` .reply((upd,bas) -> {
              // Prints to console
              System.out.println("I'm in a reply!");
              silent.send("It's been nice playing with you!", bas.getMessage().getChatId());`
